dashboard/src/containers/RoutesContainer/Routes.test.tsx

```diff
@@ -83,30 +83,3 @@ it("should render a redirect to the login page (when not authenticated)", () =>
   expect(wrapper.find(NotFound)).not.toExist();
   expect(wrapper.find(Redirect).prop("to")).toEqual("/login");
 });
-
-describe("Routes depending on feature flags", () => {
```

Comment:
Test is no longer relevant since the ff was removed (and the .dive().dive().dive() no longer got to the same element.)